specification.md

```diff
@@ -1,8 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+title: Open Agent Management Protocol
+linkTitle: OpAMP
+body_class: otel-docs-spec
+github_repo: &repo https://github.com/open-telemetry/opamp-spec
+github_project_repo: *repo
+path_base_for_github_subdir:
+  from: content/en/docs/specs/opamp/index.md
+  to: specification.md
+spelling: cSpell:ignore bitmask Fluentd varint opamp
+--->
+
 # OpAMP: Open Agent Management Protocol
```

Comment:
@tigrannajaryan et al.: could we drop the `OpAMP: ` prefix to this title? On the website, OpAMP will be in the breadcrumbs; and the abbreviation is introduced in the first paragraph below.

Reply:
Yes, I think we can do it.

Reply:
Done ✔️